backends/gpu/lib/kernels/blas_kernels.cc

```diff
@@ -200,7 +200,7 @@ static Error BlasTrsmBatch(
     const T** a_array = const_cast<const T**>(b_array + batchCount);
 
     auto side_mode = wrapper::BlasSideMode::FromOpaqueValue(*sideMode);
-    ptrdiff_t a_batch_stride = side_mode == CUBLAS_SIDE_LEFT ? m * m : n * n;
+    ptrdiff_t a_batch_stride = (side_mode == rocblas_side_left) ? m * m : n * n;
```

Comment:
I think this can be `size_mode == CUBLAS_SIDE_LEFT || size_mode == rocblas_size_left`